contracts/core/TribeAuth.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import "../fei/IFei.sol";
+import "./TribeRoles.sol";
+import "../external/solmate/src/auth/Auth.sol";
+import "../external/solmate/src/auth/authorities/MultiRolesAuthority.sol";
+import "@openzeppelin/contracts/security/Pausable.sol";
+
+abstract contract TribeAuth is TribeRoles, Auth, Pausable {
+
+    IFei public immutable fei;
+
+    IERC20 public immutable tribe;
+
+    constructor(Authority _authority, IFei _fei, IERC20 _tribe) 
```

Comment:
Should we go ahead and hardcode in fei & tribe as a constant? If they're going to immutable I don't see a reason to include them as params.

Reply:
unit tests will break if you hardcode these vars